contents/articles/2017-01_image_user_auth/index.md

```diff
@@ -0,0 +1,23 @@
+--- 
+title: Personalized Image-based User Authentication using Wearable Cameras
+date: Wed Jan 11 2017 11:18:18 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Ngu Nguyen et al., propose a novel personalized user authentication mechanism that generates an image-based passwords from egocentric videos captured by wearable cameras..."
+featured_img: "../../../../media/images/blog/.jpg"
+featured_img_thumb: "../../../../media/images/blog/.jpg"
+---
+
+<img src="../../../../media/images/blog/.jpg" class='Feature-image' alt="">
+
+<div class="small u-padBottom--2">Image Source: [Personalized Image-based User Authentication using Wearable Cameras](https://arxiv.org/pdf/1612.06209.pdf)</div>
+
+Personal devices are a part of our everyday life that operate and store users private data. Such devices face an array of threats that have become a primary concern for users. The existing authentication mechanism requires users to register and remember one static password for each device at a time. However static passwords are vulnerable to various methods of attacks. 
+
+[Ngu Nguyen](http://ambientintelligence.aalto.fi/team.html) et al., propose a novel personalized user authentication mechanism that generates an image-based passwords from egocentric videos captured by wearable cameras. 
+
+One of the process of generating authentication challenges uses the [Pupil headset](https://pupil-labs.com/store/#pupil) to capture egocentric videos and eye fixations to select key frames that are possible to discriminate. The frames are then split into segments and grouped to form fresh and personalized image-based user authentication mechanism. The image-based mechanisms are evaluated to be comparable to static approaches while decreasing security vulnerabilities.
```

Comment:
Explicitly discuss the role of eye tracking in this project. You may also want to link to one of their references from 2015